test/e2e/upgrades/sigapps/BUILD

```diff
@@ -0,0 +1,56 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_library",
+)
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "daemonsets.go",
+        "deployments.go",
+        "job.go",
+        "statefulset.go",
+    ],
+    tags = ["automanaged"],
+    deps = [
+        "//pkg/api/v1/helper:go_default_library",
```

Comment:
@foxish
Some of the dependencies below may be unnecessary. Is there a way to generate BUILD files automatically? For now, I manually modify the BUILD files. I am looking at direction of `go build` and `hack/update-bazel.sh`.